library/python/envoy_mobile/BUILD

```diff
@@ -0,0 +1,49 @@
+load("@pybind11_bazel//:build_defs.bzl", "pybind_extension")
```

Comment:
Having `envoy_mobile` in the path seems a little bit funky. Could this be `pybind` or `cc_platform`?

Also maybe the `cc_library` belongs in `//library/cc` and just the `pybind_extension` should go here.

Reply:
original naming scheme was just trying to match up folder name w/ how it would be imported in python. right now the `.so` built from `//library/python/envoy_mobile:envoy_mobile.so` can just be imported from python as `import envoy_mobile`, which would be canonical naming scheme 

there's nothing keeping the `.so` name and the dir name coupled, though, so i'm happy to change it to whatever